package com.project.usecases;

import java.util.List;
import java.util.Scanner;

import com.project.bean.Problem;
import com.project.deo.ProblemDao;
import com.project.deo.ProblemDaoImpl;
import com.project.exceptions.EngineerException;
import com.project.exceptions.ProblemException;

public class GetListOfEmployeeAssignProblemUseCase {

	public static void main(String[] args) {
	
//		Scanner sc=new Scanner(System.in);
//		System.out.println("Enter here problem name to see the status... ");
//		String pname = sc.next();
//		
//         ProblemDao dao10=new ProblemDaoImpl();	
//         
//         String result;
//  	   
//     	try {
//     		result = dao10.getAssignProblemList(pname);
//     		System.out.println(result);
//     	} catch (proException e) {
//     		// TODO Auto-generated catch block
//     		e.printStackTrace();
//     	}
//         
//		try {
//			
//			
//			
//			
//			List<Problem> problems=dao10.getAssignProblemList();
//			problems.forEach(e -> {
//				
//				System.out.println("Problem id is : "+e.getPid());
//				System.out.println("Problem name is : "+e.getPname());
//				System.out.println("Problem category is : "+e.getPcategory());
//				System.out.println("Problem status is : "+e.getPstatus());
//				
//				
//				
//				System.out.println("=================================");
//				
//			});
//			
//		} catch (ProblemException e) {
//			// TODO: handle exception
//			System.out.println(e.getMessage());
//		}
		
		

	}

}
